gpii/node_modules/gpii-db-operation/src/DbUtils.js

```diff
@@ -36,3 +36,38 @@ gpii.dbOperation.composeError = function (error, termMap) {
 gpii.dbOperation.getCurrentTimestamp = function () {
     return new Date().toISOString();
 };
+
+/**
+ * This function looks at an item to make sure it exists properly, useful for
+ * looking at the results of a `{dataStore}.findById` or similar operation. If
+ * the item exists as it should, a promise resolved with the item is
+ * returned, otherwise a rejected promise is returned with the supplied `errorMessage`.
+ *
+ * The current criteria for validity is that the item is not undefined and has
+ * the `type` that is supplied in the arguments.
+ *
+ * @param  {Object} item         Item, mostly a return from another API or CouchDB call
```

Comment:
Follow the JSDoc convention. Replace spaces in between "item" and "Item" with "-". The same comment applies to other lines.

Reply:
This should be fixed now.